src/obsidian_to_org/__main__.py

```diff
@@ -172,4 +194,4 @@ def convert_directory():
 
 
 if __name__ == "__main__":
-    main()
+    convert_directory()
```

Comment:
**Question:** What's the motivation for this change? It seems unrelated to copying over images.

Reply:
Hi @jml . I didn't see `main` function in your script, so I have changed the script so that `convert_directory` would be called on start.

Reply:
That makes sense, thanks.